xbmc/cores/VideoPlayer/DVDInputStreams/DVDFactoryInputStream.cpp

```diff
@@ -89,7 +94,9 @@ std::shared_ptr<CDVDInputStream> CDVDFactoryInputStream::CreateInputStream(IVide
   else if (URIUtils::IsPVRRecording(file))
     return std::shared_ptr<CInputStreamPVRRecording>(new CInputStreamPVRRecording(pPlayer, fileitem));
 #ifdef HAVE_LIBBLURAY
-  else if (fileitem.IsType(".bdmv") || fileitem.IsType(".mpls") || StringUtils::StartsWithNoCase(file, "bluray:"))
+  else if (fileitem.IsType(".bdmv") || fileitem.IsType(".mpls") 
```

Comment:
Can this be turned into a function call with a meaningful name instead of grouping too many  `or` conditions? Something like `fileitem.IsDiskInformationFile()` or whatever best describes the referred file extensions?

Reply:
I'd like to left this to some dev having more global view of the code.